tsdb/meta.go

```diff
@@ -1329,11 +1309,16 @@ func NewSeries(key string, tags map[string]string) *Series {
 }
```

Comment:
Is there a lot of read contention on `Series.shardIDs`? If there's not then it would be more efficient to use a `sync.Mutex`.